v2/pages/userInstanceQuestionExam/userInstanceQuestionExam.ejs

```diff
@@ -50,29 +50,23 @@
             
             <% if (showAllSubmissions) { %>
             <div class="panel-group">
-                <% for (sNum = 0; sNum < allSubmissions.length; sNum++) { 
-                thisSubmission = allSubmissions[sNum]; 
-                if (thisSubmission.correct === null) {
-                    panelType = "panel-info"; 
-                    submissionLabel = "<span class='label label-primary'>saved, not graded</span>"; 
-                }
-                else if (thisSubmission.correct) {
-                    panelType = "panel-success";
-                    submissionLabel = "<span class='label label-success'>correct</span>";
-                }
-                else {
-                    panelType = "panel-danger";
-                    submissionLabel = "<span class='label label-danger'>incorrect</span>";
-                } %>
-                <div class="panel <%= panelType %> submission-block">
+                <% allSubmissions.forEach(function (thisSubmission, sNum, allSubmissions) {
```

Comment:
No need for the final `allSubmissions` argument to the `function()`. Also don't leave a space after `function`.